app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```
@@ -65,13 +69,16 @@
 }
 
 .gem-c-feedback__prompt-question {
+  vertical-align: text-top;
   display: inline-block;
 
   // There's a global h3 rule in some layouts that interferes with this component
   margin: 0;
 
   margin-left: govuk-spacing(4);
   margin-right: govuk-spacing(4);
+  margin-top: govuk-spacing(2);
+  margin-bottom: govuk-spacing(2);
```

Can these margins be put onto one line?
```suggestion
  margin: govuk-spacing(2) govuk-spacing(4);
```